spec/integrity/common_mark_spec.rb

```diff
@@ -34,6 +34,8 @@
         cooked.gsub!(" class=\"lang-auto\"", '')
         cooked.gsub!(/<span class="hashtag">(.*)<\/span>/, "\\1")
         cooked.gsub!(/<a name="(.*)" class="anchor" href="#\1*"><\/a>/, "")
+        # we support data-attributes which is not in the spec
+        cooked.gsub!("<pre data-startline=\"3\">", '<pre>')
```

Comment:
We are literally not following the code spec here so it's expected to have to do this